azext_iot/iothub/params.py

```diff
@@ -297,15 +297,16 @@ def load_iothub_arguments(self, _):
         context.argument(
             "resource_group_name",
             options_list=["--destination-resource-group", "--dg"],
-            help="Name of resource group of the IoT Hub to which the origin hub state will be copied to.",
+            help="Name of resource group of the IoT Hub to which the origin hub state will be copied to."
+            "If not provided, will use the origin IoT Hub's resource group.",
             arg_group="IoT Hub Identifier"
         )
         context.argument(
             "login",
             options_list=["--destination-hub-login", "--dl"],
             validator=mode2_iot_login_handler,
             help="This command supports an entity connection string with rights to perform action on the destination hub. "
-            'Use to avoid session login via "az login" for this IoT Hub instance. '
+            "Use to avoid session login via \"az login\" for this IoT Hub instance. "
```

Comment:
One alternative to consider for the dbl quotes is using the back tick ` around az login, which will render as the code view syntax in docs, and still have a "quote effect" locally.

Reply:
done